### ISSUE TITLE
Default doesn't work with action fields

### ISSUE THREAD

**Issue by garrettjoecox** [inspheric/nova-defaultable-fields#2]
Reproduction:
- Create a nova action
- Register the action with User resource
- Add a field using default to the action fields:
```
    public function fields()
    {
        return [
            Text::make('Message')
                ->default('default value'),
        ];
    }
```
- Run action on a user
- Observe that the default value did not populate the input.

**Comment by kitbs** [comment]
Hi @garrettjoecox

Actions are not currently supported but I had always had it in mind to do eventually.

I'd be happy to accept a PR in the meantime.

**Comment by kitbs** [comment]
Hi @garrettjoecox I have released v1.2 which supports actions.
